ui/raidboss/data/07-dt/trial/queen-eternal.ts

```diff
@@ -367,13 +380,109 @@ const triggerSet: TriggerSet<Data> = {
       outputStrings: {
         fiveAOE: {
           en: '5x AoEs',
+          de: '5x AoEs',
           cn: 'AoE (5次)',
           ko: '전체공격 (5x)',
         },
       },
     },
   ],
   timelineReplace: [
+    {
+      'locale': 'de',
+      'missingTranslations': true,
+      'replaceSync': {
+        'Queen Eternal': 'Ewig(?:e|er|es|en) Königin',
+      },
+      'replaceText': {
+        '\\(abandonment\\)': '()',
```

Comment:
I think you can add a proper translation here: this is the same skill as queen-eternal-ex's (Dorito Stack).
https://github.com/OverlayPlugin/cactbot/blob/8f90e01f383c55eeb199838002d332d5a8294cdd/ui/raidboss/data/07-dt/trial/queen-eternal-ex.ts#L732-L734

![zen_wn8EB6RM1i](https://github.com/user-attachments/assets/7a721581-d387-404f-86fa-5abb785948bd)
https://youtu.be/zzkMb0ElOyU?si=1LkBJa90xsHVZnf2&t=215